core/src/test/java/hudson/UtilTest.java

```diff
@@ -676,7 +676,7 @@ public void createDirectoriesInRoot() throws Exception {
             assertEquals(newDirInRoot.resolve("new1"), Util.createDirectories(newDirInRoot.resolve("new1")).toRealPath());
             Util.createSymlink(newSymlinkInRoot.getParent().toFile(), newDirInRoot.getFileName().toString(), newSymlinkInRoot.getFileName().toString(), TaskListener.NULL);
             assertEquals(newDirInRoot.resolve("new2"), Util.createDirectories(newSymlinkInRoot.resolve("new2")).toRealPath());
-        } catch (AccessDeniedException e) {
+        } catch (FileSystemException e) {
```

Comment:
Could also adjust the `assumeFalse` a few lines above if we don't want to change this.

This is a superclass of `AccessDeniedException`